lib/tokens/token.js

```diff
@@ -41,41 +41,13 @@ module.exports = function (log, config, random, P, hkdf, Bundle, error) {
     this.createdAt = details.createdAt || 0
```

Comment:
Most recently, because the API test client in ./tests/client/ calls `fromHex` without specifying the `createdAt` time, so we gotta default it to something, and zero is much safer than e.g. the current time.

Of course the right answer might be to fix those damn tests to use the API correctly, or to use a simpler wrapper around it.